### ISSUE TITLE
remove branching for d3 projections?

### ISSUE THREAD

**Issue by Fil** [observablehq/plot#1134]
To simplify the number of branches in the projection code, it seems that we could safely remove this path:
https://github.com/observablehq/plot/blob/4ee796d74e1c88163824f358f2f922639c3df502/src/projection.js#L33

support for, say, d3.geoAirocean would change slightly:
~~~diff
- projection: d3.geoAirocean().scale(20)
+ projection: () => d3.geoAirocean().scale(20)
~~~

**Comment by mbostock** [comment]
No, because then we would be mutating the projection instance owned by the user.
